cime/src/drivers/mct/shr/seq_flds_mod.F90

```diff
@@ -2394,6 +2395,148 @@ subroutine seq_flds_set(nmlfile, ID, infodata)
     call set_glc_elevclass_field(name, attname, longname, stdname, units, x2l_fluxes_from_glc, &
          additional_list = .true.)
 
+     name = 'So_blt'
```

Comment:
This block of added fields should be enclosed in a conditional, `if (trim(cime_model) == 'e3sm') then`

Reply:
Fixed in 59a26f5ec4f1598113a3e68d7f7498c8fa108005